src/mds/Server.cc

```diff
@@ -1143,11 +1143,31 @@ void Server::reply_client_request(MDRequestRef& mdr, MClientReply *reply)
   // clean up request
   mdcache->request_finish(mdr);
 
+  dout(10) << __func__ << ": finished request, examining tracei" << dendl;
+  if (tracei) {
+    dout(10) << "  tracei: " << *tracei << dendl;
+    if (tracei->get_parent_dn()) {
+      dout(10) << "  tracei parent dn: " << *(tracei->get_parent_dn()) << dendl;
+      dout(10) << "  tracei is remote: " << tracei->get_parent_dn()->get_projected_linkage()->is_remote() << dendl;
+
+    } else {
+      dout(10) << "  tracei no parent dn" << dendl;
+    }
+  } else {
+    dout(10) << "  tracei not set" << dendl;
+  }
+
+  if (tracedn) {
+    dout(10) << "  tracedn: " << *tracedn << dendl;
+    dout(10) << "  tracedn remote: " << tracedn->get_projected_linkage()->is_remote() << dendl;
+  }
```

Comment:
Ah, this stuff is from debugging the eval_remote path.  Yeah, it's redundant now, removed.